doc/fwutil/fwutil.md

```diff
@@ -462,6 +519,321 @@ Aborted!
 
 **Note:** the default option is _--image=current_
 
+#### 2.2.2.4 Auto-update commands
+
+##### 2.2.2.4.1 Overview
```

Comment:
@sujinmkang in case of boot_action=any what should be the expected behaviour if platform component supports both cold and fast reboot? Do we expect platform API will add this item to both cold_* and fast_* task lists?

Reply:
As I replied in your previous question, we need to limit the boot action by none/fast/warm/cold. And in the previous comments, we don't support multiple firmware auto update for the same or other boot type.